### ISSUE TITLE
nullpointer when creating zip based on folder

### ISSUE THREAD

**Issue by alebo611** [srikanth-lingala/zip4j#92]
(version 2.2.2) This code:
```
ZipFile zipFile = new ZipFile("hello.zip");
zipFile.addFolder(new File("myfolder"));
```
will cause a nullpointer:
```
Caused by: java.lang.NullPointerException
	at net.lingala.zip4j.tasks.AddFolderToZipTask.setDefaultFolderPath(AddFolderToZipTask.java:49)
	at net.lingala.zip4j.tasks.AddFolderToZipTask.executeTask(AddFolderToZipTask.java:28)
	at net.lingala.zip4j.tasks.AddFolderToZipTask.executeTask(AddFolderToZipTask.java:17)
	at net.lingala.zip4j.tasks.AsyncZipTask.performTaskWithErrorHandling(AsyncZipTask.java:42)
```

**Comment by srikanth-lingala** [comment]
Issue fixed. Will include the fix in the next release.

**Comment by srikanth-lingala** [comment]
Fixed in v2.2.4 released today

**Comment by alebo611** [comment]
Great news :) !

Den mån 4 nov. 2019 kl 21:57 skrev Srikanth Reddy Lingala <
notifications@github.com>:
